### PR TITLE
Updated README.rst 

### PR DESCRIPTION
**Pull request by cemurray3**
Included information about contributing based on text in README.rst from dust_extinction.

### CODE REVIEW

**Comment by karllark** [comment]
This is great.  I did notice that there is some documentation on the datamodel.py file in beast_setup.rst.  Can you take a look and see if there is a merge of your new material and that file?

**Comment by karllark** [comment]
Addresses #119.

File: docs/workflow.rst [hunk 1]
```diff
@@ -239,6 +237,7 @@ where the filebase where it is the first portion of the output stats filenames
 Reorganize the results into spatial region files
 ================================================
   
+<<<<<<< HEAD
```
Comment (karllark) [inline docs/workflow.rst]: This looks like a comment leftover from a merge conflict.  Please check.

File: beast/plotting/plot_indiv_fit.py [hunk 2]
```diff
@@ -332,7 +332,8 @@ def plot_beast_ifit(filters, waves, stats, pdf1d_hdu):
     starnum = args.starnum
 
     # base filename
-    filebase = args.filebase
+    #filebase = args.filebase
+    filebase = '/Users/clairemurray/beast/beast/examples/phat_small/beast_example_phat/beast_example_phat'
```
Comment (karllark) [inline beast/plotting/plot_indiv_fit.py]: This is a temporary override.  Should be deleted and the previous line uncommented.

**Comment by karllark** [comment]
There is an issue in workflow.rst.  The travis ci test for building the documentation "fails" with:

> /home/travis/build/BEAST-Fitting/beast/docs/datamodel.rst:153: WARNING: Literal block ends without a blank line; unexpected unindent.

**Comment by cemurray3** [comment]
I think I fixed the problems with the erroneous text, but the build showed an error related to the coverage that I'm not sure how to fix.. sorry for the chaos! I will do a cleaner job of PRs in the future :)

**Comment by karllark** [comment]
No worries.  Thanks for your prompt attention!  And there is no need for a cleaner PR.  All PRs are good!

**Comment by karllark** [comment]
Also addresses #71.

**Pull request merged**